packages/lms/src/lib/components/header/Header.svelte

```diff
@@ -13,7 +14,7 @@
   <svelte:fragment slot="lead">
     <div class="flex items-center">
       <button class="lg:hidden btn btn-sm mr-4" on:click={drawerOpen}>
-        <i class="icon-f">menu</i>
+        <Icon iconName="menu" />
```

Comment:
```suggestion
        <Icon name="menu" />
```
There's no reason not to use the shorter `name` property since `name` in this case isn't a reserved keyword.

Reply:
I agree.